pwiz_tools/Skyline/FileUI/PeptideSearch/BuildPeptideSearchLibraryControl.cs

```diff
@@ -281,13 +281,27 @@ public void AddSearchFiles(IEnumerable<string> fileNames)
             SearchFilenames = BuildLibraryDlg.AddInputFiles(WizardForm, SearchFilenames, fileNames, PerformDDASearch);
         }
 
+        // For test purposes, lets us test error handling
+        public string CutOffScoreText

```

Comment:
Let's implement this as an internal private _cutoffScore that starts as null (default) and only gets its double value in the Next() function with code like:

var helper = new MessageBoxHelper(this.ParentForm);
if (!helper.ValidateDecimalTextBox(textCutoff, out _cutoffScore))
    return;

public double? CutoffScore would be get only to get the _cutoffScore and CutoffScoreText could remain to set the control text.